### PR TITLE
Gutenberg Extension Compatibility

### PR DESCRIPTION
**Pull request by bordoni**
### For more information about Gutenberg checkout our [Extension](https://github.com/moderntribe/events-gutenberg)
_Ref:_  [C#101632](http://central.tri.be/issues/101632)
_Rel:_ https://github.com/moderntribe/tribe-common/pull/642

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Gutenberg.php:45](https://github.com/moderntribe/the-events-calendar/blob/f884c2cc0878e1fe14fd01bee9552de3c715af7b/src/Tribe/Gutenberg.php#L45)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Gutenberg.php:45](https://github.com/moderntribe/the-events-calendar/blob/141c4c31f13d456cee56666d09d957b44e1efaab/src/Tribe/Gutenberg.php#L45)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Gutenberg.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+			return false;
+		}
+
+		$url = 'https://theeventscalendar.com/extensions/events-gutenberg/';
+		$link = sprintf(
+			'<a href="%1$s" target="_blank">%2$s</a>',
+			esc_url( $url ),
+			esc_html__( 'check how add The Events Calendar compatibility!', 'the-events-calendar' )
+		);
+		$text = __( 'We noticed that you are using Gutenberg on this WordPress instalation, %1$s', 'the-events-calendar' );
```
Comment (ggwicz) [inline src/Tribe/Gutenberg.php]: change to `installation`

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Gutenberg.php:45](https://github.com/moderntribe/the-events-calendar/blob/1bb52f56df98ea392d84b08f043c4f28c6642803/src/Tribe/Gutenberg.php#L45)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Gutenberg.php:45](https://github.com/moderntribe/the-events-calendar/blob/5198ec6b18a2337552d42b03929c4d3c859bfbf6/src/Tribe/Gutenberg.php#L45)

> via jenkins/codesniffer-phpcs

**Comment by bordoni** [comment]
No need for changelog here George.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Gutenberg.php:45](https://github.com/moderntribe/the-events-calendar/blob/ae2dfbdfa08be1cabcde6fc289ac421c870850f5/src/Tribe/Gutenberg.php#L45)

> via jenkins/codesniffer-phpcs

**Pull request merged**